.distignore

```diff
@@ -0,0 +1,12 @@
+/.git
```

Comment:
This file supersedes https://github.com/cloudflare/Cloudflare-WordPress/blob/master/scripts/deployment/trim_code_for_release.sh and once this is merged, we can redo/cleanup the documentation to reflect the new steps.